mixer/adapter/cloudwatch/operatorconfig/cloudwatch.yaml

```diff
@@ -14,7 +14,31 @@ spec:
    response_code: response.code | 200
  monitored_resource_type: '"UNSPECIFIED"'
 ---
-# handler configuration for adapter 'metric'
+# instance configuration for template 'logentry'
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+ name: accesslog
+ namespace: istio-system
+spec:
+ severity: '"Default"'
+ timestamp: request.time
+ variables:
+  source_service: source.labels["service"] | "unknown"
```

Comment:
just curious: is 'service' label here different than 'app'?  It'd be nice to encourage some standard for this across Istio.

Reply:
changed it to 'app' to maintain standard.